tls/raw-data/raw-tls-artifacts-ha-amd64-gcp-ovn.json

```diff
@@ -5252,6 +5339,488 @@
+                "Issuer": {
+                  "CommonName": "kubelet-bootstrap-kubeconfig-signer",
+                  "SerialNumber": "",
+                  "PubkeyModulus": "",
+                  "Issuer": null
+                }
+              },
+              "SignatureAlgorithm": "SHA256-RSA",
+              "PublicKeyAlgorithm": "RSA",
+              "PublicKeyBitSize": "2048 bit",
+              "ValidityDuration": "10y",
+              "Usages": [
+                "KeyUsageDigitalSignature",
+                "KeyUsageKeyEncipherment",
+                "KeyUsageCertSign"
+              ],
+              "ExtendedUsages": []
+            },
+            {
+              "CertIdentifier": {
+                "CommonName": "openshift-kube-apiserver-operator_node-system-admin-signer@1704386391",
+                "SerialNumber": "6173699297008815851",
+                "PubkeyModulus": "21248436608359792187395410922349184612841042387927193612000558931928757920974875963771273548688297783813926148256148748755321570680059248617781042307020006648763089126674065187768389132244110636947972060442235910074879818475954156382020296801765845951011581693303847196507897279730072224635768042399174029810875644513586610386566825896897283238888820444129488926465552942785387217206817883435168145420444479290294673831879952649140971598841602103346893063127426423596167400151441925164055372284871422591772685640455898220631655375401162629498384358816726727395406756268122311786614450223196787836340049408186959193889",
+                "Issuer": {
+                  "CommonName": "openshift-kube-apiserver-operator_node-system-admin-signer@1704386391",
+                  "SerialNumber": "",
+                  "PubkeyModulus": "",
+                  "Issuer": null
+                }
+              },
+              "SignatureAlgorithm": "SHA256-RSA",
+              "PublicKeyAlgorithm": "RSA",
+              "PublicKeyBitSize": "2048 bit",
+              "ValidityDuration": "365d",
+              "Usages": [
+                "KeyUsageDigitalSignature",
+                "KeyUsageKeyEncipherment",
+                "KeyUsageCertSign"
+              ],
+              "ExtendedUsages": []
+            }
+          ]
+        },
+        "Status": {
+          "Errors": null
+        }
+      },
+      {
+        "LogicalName": "",
+        "Description": "",
+        "Name": "etcd-metric-signer",
```

Comment:
Why didn't this combine with the etcd-metric-signer in the cluster?

Reply:
We need this to combine so that we have most of these auto-discover ownership, right?